spec/activefedora/association_spec.rb

```diff
@@ -271,6 +271,31 @@ class Image < ActiveFedora::Base
     end
   end
 
+  context "with a type validation" do
+    let(:image) { Image.new }
+    let(:parent) { Image.new }
+    before do
+      class TypeValidator
+        def self.validate!(record)
+        end
+      end
+      class Image < ActiveFedora::Base
+        aggregates :files, :type_validator => TypeValidator
+      end
+      allow(TypeValidator).to receive(:validate!).with(image).and_raise ActiveFedora::AssociationTypeMismatch
+    end
+    after do
+      Object.send(:remove_const, :Image)
+      Object.send(:remove_const, :TypeValidator)
+    end
+
+    context "when an invalid item is passed" do
+      it "should raise an error" do
+        expect{parent.files += [image]}.to raise_error ActiveFedora::AssociationTypeMismatch
```

Comment:
Add spaces around braces please.  Also just using ` = [image]` rather than `+= [image]` would avoid the read operation.